src/Centreon/Infrastructure/PlatformTopology/PlatformTopologyRegisterRepositoryAPI.php

```diff
@@ -195,4 +218,115 @@ public function registerPlatformToParent(
             );
         }
     }
+
+    public function deletePlatformToParent(
+        Platform $platform,
+        PlatformInformation $platformInformation,
+        ?Proxy $proxyService = null
```

Comment:
should be named `$proxy` cause it's an entity, not a service

Reply:
indeed I also changed it into the other method RegisterPlatformToParent